dev/isort.cfg

```diff
@@ -1,9 +1,9 @@
 # Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
+# contributor license agreements. See the NOTICE file distributed with
```

Comment:
No, this is not pep8 related!

I deleted dev/tox.ini and created a new file dev/isort.cfg from scratch. But the fact is git "finds" file renames based on content similarities, so it sees this as a file rename. Actually, I did deleted the tox.ini in another commit than the commit that creates isort.ini (see https://github.com/apache/spark/pull/14567/commits), so I guess this "visualisation" of a file renaming comes from github